DXRMapFixups/DeusEx/Classes/DXRFixupM01.uc

```diff
@@ -94,6 +98,14 @@ function PreFirstEntryMapFixes()
             }
         }
 
+        if (#defined(injections) && VanillaMaps) {
+            foreach AllActors(class'#var(prefix)ComputerPublic', compublic) {
+                compublic.SetLocation(vectm(741.36, 1609.34, 289.27));
+                compublic.SetRotation(rotm(0, -16384, 0, 0));
```

Comment:
All of these rotm's should be using GetRotationOffset instead of setting 0 (especially since GetRotationOffset would return 16384)